ember-can/src/-private/resolver.ts

```diff
@@ -0,0 +1,12 @@
+import type Resolver from 'ember-resolver';
+
+export default function extendResolver(
+  resolver: typeof Resolver,
+): typeof Resolver {
+  return class EmberCanResolver extends resolver {
```

Comment:
Would this part not also work with earlier resolver versions? I think I would prefer a breaking release and doing this for everyone over conditionally using `reopen`.

Reply:
@RobbieTheWagner yes it works also with old versions... okay i will change it so that is braking

Reply:
@RobbieTheWagner done, changes are now braking